### ISSUE TITLE
Vector Drawables Pre-Lollipop

### ISSUE THREAD

**Issue by lawloretienne** [lisawray/passwordview#14]
So I always thought that Vector Drawables were only supported on API 21+.  But, I cloned your repo and ran your app on a API 19 device and the Vector Drawable rendered perfectly fine.  Is there some reason why this all works on pre-lollipop?  I was hoping you could explain.  

Also in your `PasswordView.java` class, on lines 67 and 68 you have 

``` java
eye = ContextCompat.getDrawable(getContext(), R.drawable.ic_eye).mutate();
eyeWithStrike = ContextCompat.getDrawable(getContext(), R.drawable.ic_eye_strike).mutate();
```

Shouldn't that be replaced with 

``` java
eye = VectorDrawableCompat.create(getResources(), R.drawable.ic_eye, null).mutate();
eyeWithStrike = VectorDrawableCompat.create(getResources(), R.drawable.ic_eye_strike, null).mutate();
```

to support vector drawables on all API versions?

**Comment by behelit** [comment]
It seems to work even on API 17, without making this change

**Comment by lisawray** [comment]
Hey!  This library was based on an earlier version of the support library, maybe the first one providing vector drawable support, where vector drawables "auto-magically" worked.  However, this vector support was reverted in a following version, due to memory problems in the implementation. 

I've just updated to the latest support library, where you definitely need to use VectorDrawableCompat. :) https://github.com/lisawray/passwordview/commit/42b95efe5709edbbffe94d8ccb6a6a460302d3ca
